packages/core/src/dialects/abstract/query-generator-typescript.ts

```diff
@@ -24,8 +24,10 @@ export interface QueryGeneratorOptions {
   dialect: AbstractDialect;
 }
 
+// DO NOT MAKE THIS CLASS PUBLIC!
 /**
- * Temporary class to ease the TypeScript migration
+ * This is a temporary class used to progressively migrate the AbstractQueryGenerator class to TypeScript by slowly moving its functions here.
```

Comment:
This also shows up on `AbstractQueryGenerator` so I think we'll need to overwrite it there. That's what we do with `AbstractQueryInterface` so this issue is not occuring there